scripts/travis/Jenkinsfile

```diff
@@ -18,6 +25,11 @@ def buildClosures(arg) {
     }
     closures
 }
+
+# This is where the Jenkinsfile gets called and does its stuff
+# It's separated out into clang and gcc tests that get called 
+# sequentially.
+
 node {
     cleanWs()
     checkout scm
```

Comment:
Where do the names that show up in the Gui come from, is it here?

Reply:
The names that show up in the gui are from the .travis.yml test matrix.